ktxspec.adoc

```diff
@@ -877,14 +877,25 @@ _dfDescriptorBlock_.
 * If `<<_vkformat,vkFormat>>` is not `VK_FORMAT_UNDEFINED`, the DFD's
   `texelBlockDimension*`, `bytesPlane*` and sample information fields
   must match the format's definition. The `colorModel` must be
-  `KHR_DF_MODEL_RGBSDA` or the matching block compressed color model
-  listed in <<KDF13>> Section 5.6 or its successors, currently
-  `KHR_DF_MODEL_BC1A` to `KHR_DF_MODEL_UASTC`.
+  `KHR_DF_MODEL_RGBSDA`, `KHR_DF_MODEL_YUVSDA` or the matching block
+  compressed color model listed in <<KDF13>> Section 5.6 or its
+  successors, currently `KHR_DF_MODEL_BC1A` to `KHR_DF_MODEL_UASTC`.
+  `KHR_DF_MODEL_YUVSDA` should be used for all non-prohibited
+  `+*_422_*+` formats, `KHR_DF_MODEL_RGBSDA` for the rest.
```

Comment:
The latest addition sounds ambiguous as it's not clear what "the rest" means: prohibited formats or non-422 formats. In any case, it's redundant since the previous sentence already limits the allowed color models to RGB, YUV, and compressed.